person.rb

```diff
@@ -1,9 +1,13 @@
-class Person
+require './nameable'
+require 'securerandom'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
-  def initialize(id, age, name = 'Unknown', parent_permission = true) # rubocop:disable Style/OptionalBooleanParameter
-    @id = id
+  def initialize(age, name = 'Unknown', parent_permission = true) # rubocop:disable Style/OptionalBooleanParameter
+    super()
```

Comment:
- [optional] Kindly ensure that no linter is disabled, linters are the to catch bugs ahead to improve our project. ℹ️